crates/circuit/src/circuit_data.rs

```diff
@@ -1394,13 +1394,7 @@ impl CircuitData {
                                 // Standard gates can all rebuild their definitions, so if the
                                 // cached py_op exists, update the `params` attribute and clear out
                                 // any existing cache.
```

Comment:
Should we update this comment and perhaps add a GitHub link to the relevant issue?

Reply:
Sure!